.github/workflows/gradle.yml

```diff
@@ -0,0 +1,21 @@
+name: Build cadence-java-samples
+
+on: [ push ]
```

Comment:
let's run it on pull requests as well so you can validate it works
`on: [push, pull_request]`